crates/rpc-engine-types/src/payload.rs

```diff
@@ -117,6 +117,9 @@ pub struct ExecutionPayloadEnvelopeV3 {
     /// Introduced in V3, this represents a suggestion from the execution layer if the payload
     /// should be used instead of an externally provided one.
     pub should_override_builder: bool,
+    /// Optimism-specific - Ecotone parent beacon block root
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub parent_beacon_block_root: Option<B256>,
```

Comment:
I'm fine with that for now, if you create a new reth issue for better rpc execution payload customization

cc @rjected

Reply:
is this fixed with the customized engine types in node builder?

Reply:
not yet https://github.com/paradigmxyz/reth/pull/6871